pkg/controller/jenkins/configuration/user/seedjobs/seedjobs.go

```diff
@@ -303,7 +316,7 @@ func (s SeedJobs) createAgent(jenkinsClient jenkinsclient.Jenkins, k8sClient cli
 
 	// Create node if not exists
 	if err != nil && err.Error() == "No node found" {
-		_, err = jenkinsClient.CreateNode(agentName, 1, "The jenkins-operator generated agent", "/home/jenkins", agentName)
+		_, err = jenkinsClient.CreateNode(agentName, jenkinsManifest.Spec.Master.SeedJobAgentExecutors, "The jenkins-operator generated agent", "/home/jenkins", agentName)
```

Comment:
message shoulbe something like: "A seed jobs agent owned by the Jenkins operator"